storage/testdb/testdb.go

```diff
@@ -34,14 +34,35 @@ import (
 	_ "github.com/go-sql-driver/mysql" // mysql driver
 )
 
+const (
+	// MySQLURIEnv is the name of the ENV var checked for the test mysql instance URI to use.
+	MySQLURIEnv = "MYSQL_URI"
+
+	defaultMySQLURI = "root@tcp(127.0.0.1)/"
+)
+
 var (
-	trillianSQL   = testonly.RelativeToPackage("../mysql/schema/storage.sql")
-	dataSourceURI = flag.String("test_mysql_uri", "root@tcp(127.0.0.1)/", "The MySQL uri to use when running tests")
```

Comment:
Can you double check that there aren't any internal/external users of this deleted flag?

Reply:
Alternatively, don't delete this flag, but in `mysqlURI()` method assert that only one of `--test_mysql_uri` flag or `TEST_MYSQL_URI` env is used.

Reply:
I can't check completely, but searching for the flag name comes up empty.
I'll let you and @mhutchinson discuss the relative merits of flag &&/|| ENV :)